cli/state.go

```diff
@@ -1868,3 +1826,144 @@ var stateNtwkVersionCmd = &cli.Command{
 		return nil
 	},
 }
+
+var stateDealCmd = &cli.Command{
+	Name:  "deal",
+	Usage: "Get information about deal(s)",
+	Subcommands: []*cli.Command{
+		stateGetDealSetCmd,
+		stateDealFeesCmd,
+	},
+}
+
+var stateGetDealSetCmd = &cli.Command{
+	Name:      "get-deal",
```

Comment:
`state deal get-deal`? I'd make it just `state deal get`.

Reply:
yeah, that's better